pkg/test/resourcefixture/testdata/basic/container/v1beta1/containernodepool/_generated_object_containernodepool.golden.yaml

```diff
@@ -29,6 +29,10 @@ spec:
   nodeConfig:
     diskSizeGb: 100
     diskType: pd-balanced
+    effectiveTaints:
```

Comment:
It looks  like maybe this (1) doesn't update and (2) might be a terraform thing, not a GKE thing.   Should we include it?

Reply:
I think you are right. I will test without this. This is the terraform specific field that is read only.